### PR TITLE
DIT-73 Update content on unauthorised page + remove supporting info limit

### PR DESCRIPTION
**Pull request by ed0906**

### CODE REVIEW

File: app/forms/SupportingInformationDetailsFormProvider.scala [hunk 1]
```diff
@@ -16,17 +16,15 @@
...
 import forms.mappings.Mappings
+import javax.inject.Inject
 import play.api.data.Form
 
 class SupportingInformationDetailsFormProvider @Inject() extends Mappings {
 
   def apply(): Form[String] =
     Form(
       "value" -> text("supportingInformationDetails.error.required")
-        .verifying(maxLength(100, "supportingInformationDetails.error.length"))
```
Comment (SteveSugden) [inline app/forms/SupportingInformationDetailsFormProvider.scala]: whats this change for?

Comment (SteveSugden) [reply app/forms/SupportingInformationDetailsFormProvider.scala]: Ah - the 2nd part of your PR title I guess

Comment (ed0906) [reply app/forms/SupportingInformationDetailsFormProvider.scala]: Aimee requested it the other day, I forgot to do a PR for it

Comment (ed0906) [reply app/forms/SupportingInformationDetailsFormProvider.scala]: Bit of feature creep I apologise :)

**Comment by ed0906** [comment]
Pushed to master as well by accident 👎

**Comment by SteveSugden** [comment]
As a punishment, can you fix that failing AT?

**Pull request closed without merging**